src/features/chronicle/components/ChronicleCard/ChronicleCard.tsx

```diff
@@ -25,6 +30,29 @@ export const ChronicleCard: FunctionComponent<Props> = ({
   cardWidth,
   children,
 }) => {
+  const theme = useTheme()
+
+  const [currentNumberOfLines, setCurrentNumberOfLines] = useState<number | undefined>(undefined)
+  const [shouldDisplayButton, setShouldDisplayButton] = useState(false)
+
+  // height depending on the platform
+  const DEFAULT_HEIGHT_WEB =
+    parseFloat(theme.designSystem.typography.bodyAccentS.lineHeight) * MAX_LINES * REM_TO_PX
+  const DEFAULT_HEIGHT_MOBILE =
+    parseFloat(theme.designSystem.typography.bodyAccentS.lineHeight) * MAX_LINES
+  const getDefaultHeight = Platform.OS === 'web' ? DEFAULT_HEIGHT_WEB : DEFAULT_HEIGHT_MOBILE
```

Comment:
```suggestion
  const defaultHeight = Platform.OS === 'web' ? DEFAULT_HEIGHT_WEB : DEFAULT_HEIGHT_MOBILE
```
J'aurais juste renommer, `getDefaultHeight` ça me donne l'impression que c'est une fonction